lib/abort-request.js

```diff
@@ -0,0 +1,8 @@
+var noop = require("./util/noop")
```

Comment:
Can you change this as well to `noop.js`?

Reply:
I can, although half the library uses this Node convention of not appending the module file extension to the require call -- we should probably standardise this one way or the other -- something for #97?

Reply:
You're right; I didn't notice that.

I guess leave it for now, and make a comment in #97.